include/effects/matrix/PatternBounce.h

```diff
@@ -58,6 +58,8 @@
 #include "Boid.h"
 #include "ledstripeffect.h"
 
+#if MATRIX_HEIGHT > 1
```

Comment:
I didn't want to introduce yet-another-config variables called "USE_BOIDS", so I just assume you can use Boids, and will initialize them, if you have a 2D matrix.  If you have a 1D strip, no Boids.